lte/gateway/c/oai/lib/pcef/pcef_handlers.cpp

```diff
@@ -92,12 +125,8 @@ void pcef_create_session(
   auto ip_str   = std::string(ip);
   // Change ip to spgw_ip. Get it from sgw_app_t sgw_app;
   magma::LocalCreateSessionRequest sreq;
-
-  sreq.mutable_sid()->set_id("IMSI" + imsi_str);
-  sreq.set_rat_type(magma::RATType::TGPP_LTE);
-  sreq.set_ue_ipv4(ip_str);
-  sreq.set_bearer_id(session_request.eps_bearer_id);
-  pcef_fill_create_session_req(session_data, &sreq);
+  pcef_fill_create_session_req(
+      imsi_str, ip_str, session_request.eps_bearer_id, session_data, &sreq);
```

Comment:
We can remove LocalCreateSessionRequest from the arguments of `pcef_fill_create_session_req` as it seems it is no longer needed. 

Also it makes sense to include imsi, ip and eps_bearer_id in the session data and simplify the signature of `pcef_create_session`. We can do this in another diff, but let's put a comment on top of the function so we don't forget about it.

Reply:
The `LocalCreateSessionRequest` is still used as the legacy fields have not been deprecated yet. I will remove them once SessionD migrates to using the bundled fields. 
I will leave a comment about the session data signature. :)

Reply:
my bad. ignore the first part of my comment.